docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -19,29 +23,35 @@ This is similar to setting `--channel=latest/stable` or shorthand `--stable` and
 To install a specific version with long term support (e.g. 2.1), or to install a beta or development release, refer to the store page for the snap, choose install, and then pick the desired channel.
 The store page also provides instructions for installation on different Linux distributions as well as the list of supported CPU architectures.
 
-For the list of EdgeX snaps, please refer [here](#edgex-snaps).
-
 ### Configuration
 [configuration]: #configuration
 
 EdgeX snaps are packaged with default service configuration files. In certain cases, few configuration fields are overridden within the snap for snap-specific deployment requirements.
 
+There are a few ways to configure snapped services. In simple cases, it should be sufficient to modify the default config files before starting the services for the first time and use config overrides to change supported settings afterwards. Please refer below to learn about the different configuration methods.
 
 #### Config files
-The default configuration files are typically placed at `/var/snap/<snap>/current/config`. Upon startup, the server configurations files are uploaded to the registry by default. Once the service starts without errors, the local configurations become obsolete and will no longer be read. Any modifications after the initial startup will not be applied. 
+The default configuration files are typically placed at `/var/snap/<snap>/current/config`. Upon a successful startup of an EdgeX service, the server configuration file (typically named `configuration.toml`) is uploaded to the Registry by default. After that, the local server configuration file will no longer be read and any modifications will not be applied. At this point, the configurations can be only changed via the Registry or by setting environment variables. Refer to [config registry](#config-registry) or [config overrides](#config-overrides) for details.
```

Comment:
How about adding one of these links for `Registry`?
https://docs.edgexfoundry.org/2.2/api/core/Ch-APICoreConfigurationAndRegistry/#service-registry
https://docs.edgexfoundry.org/2.2/security/Ch-Secure-Consul/

Reply:
Those links are for API and securing Consul. Added a hyperlink to https://docs.edgexfoundry.org/2.2/microservices/configuration/ConfigurationAndRegistry/#registry-provider